1-Draft/RFC00XX-Native-Command-Error-Handling.md

```diff
@@ -0,0 +1,219 @@
+---
+RFC: RFC00XX
+Author: Jason Helmick
+Status: Draft
+Area: Core
+Comments Due: 10/31/2020
+---
+
+# Native Command Error Handling
```

Comment:
I'd like to see the PS team dogfood the implementation of this feature by using it instead of the Start-NativeExecution function used in the PS build module.

Reply:
This was one of the motivations of doing this feature so we can remove that helper function